package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "docker-node-service",
+  "version": "0.1.0",
+  "description": "[![](http://dockeri.co/image/casestack/node-service)](https://hub.docker.com/r/casestack/node-service/)",
```

Comment:
Since we are not publishing to npm as a package, most of these are unnecessary: main, repository, author, license, bugs, homepage.

I'd be more comfortable if we lock the mustache version to `2.3.0` or `2.3.x`.